code/controllers/subsystems/mapping_vr.dm

```diff
@@ -88,6 +89,15 @@ SUBSYSTEM_DEF(mapping)
 			else
 				MT.load_new_z(centered = FALSE)
 
+/datum/controller/subsystem/mapping/proc/preloadShelterTemplates()
```

Comment:
Why is this necessary? Don't these already get loaded into `global.map_templates`?